src/style.css

```diff
@@ -0,0 +1,160 @@
+/* #region block: cards */
+
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  justify-content: center;
+
+  width: 198px;
+  height: 404px;
+
+  background: #fff;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  padding: 33px 19px 0;
+}
+
+.card__name {
+  margin: 40px 16px 4px;
```

Comment:
you could just set the paddings for the whole card block 'coz its the same everywhere and use only margin-bottom for its elements)